build.gradle

```diff
@@ -39,7 +41,7 @@ buildscript {
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.6.0'
+    classpath 'com.android.tools.build:gradle:4.0.0-beta02'
```

Comment:
Is there any reason to stay on bleeding edge? :)

I remember there was a need to use beta version because stable one did not support module-ing at that time. If that's not a case, I'd recommend staying with the stable releases.

Reply:
there's a 4.1 canary 2 out so this isn't *very* bleeding edge 😂